tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -19,8 +19,9 @@ def __init__(self, marionette):
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
+        self.manifest_url_app = None
         self.entry_point = hasattr(self, 'entry_point') and self.entry_point or None
+        DEFAULT_APP_HOSTNAME = 'gaiamobile.org'
```

Comment:
This needs to be self.DEFAULT_APP_HOSTNAME , no? 
Or perhaps just move it before __init__?